README.md

```diff
@@ -125,6 +125,107 @@ function @ 0x4011C0
 ...
 ```
 
+Additionally, capa also supports analyzing sandbox reports for identifying capabilities
+from dynamic execution traces. This is especially relevant for samples that are either packed or
+obfuscated, since static analysis on those samples would likely not yield the desired information.
+
+Currently, capa only supports reports from the [CAPE](https://github.com/kevoreilly/CAPEv2) sandbox. 
+Here's an example run on sample 05be49819139a3fdcdbddbdefd298398779521f3d68daa25275cc77508e42310 which is packed:
+
+```yaml
+$ capa 05be49819139a3fdcdbddbdefd298398779521f3d68daa25275cc77508e42310.exe
+WARNING:capa.capabilities.common:--------------------------------------------------------------------------------
+WARNING:capa.capabilities.common: This sample appears to be packed.
+WARNING:capa.capabilities.common: 
+WARNING:capa.capabilities.common: Packed samples have often been obfuscated to hide their logic.
+WARNING:capa.capabilities.common: capa cannot handle obfuscation well. This means the results may be misleading or incomplete.
+WARNING:capa.capabilities.common: If possible, you should try to unpack this input file before analyzing it with capa.
+WARNING:capa.capabilities.common: 
+WARNING:capa.capabilities.common: Identified via rule: (internal) packer file limitation
+WARNING:capa.capabilities.common: 
+WARNING:capa.capabilities.common: Use -v or -vv if you really want to see the capabilities identified by capa.
+WARNING:capa.capabilities.common:--------------------------------------------------------------------------------
+
+$ capa 05be49819139a3fdcdbddbdefd298398779521f3d68daa25275cc77508e42310.json
```

Comment:
Should we explain / mention here how the JSON was / can be obtained?

Reply:
I agree. The sample could be found on malshare I believe but the CAPE reports needs to be pointed to. I'll add hrefs to those.